### PR TITLE
Don't require starting from project directory

### PR DESCRIPTION
**Pull request by kriomant**
Seems like pull request #18 stalled, so here is my version of fix for issue #17.

### CODE REVIEW

File: src/main/scala/com/typesafe/sbt/SbtStartScript.scala [hunk 1]
```diff
@@ -233,25 +236,25 @@ object SbtStartScript extends Plugin {
...
-    exit 1
-}
-test -x '@RELATIVE_SCRIPT@' || die "'@RELATIVE_SCRIPT@' not found, this script must be run from the project base directory"
-"""
-        val part = renderTemplate(template, Map("RELATIVE_SCRIPT" -> relativeScript.toString))
-        otherFile.foldLeft(part)({ (firstPart, file) =>
-            firstPart + renderTemplate("""test -e '@OTHER_FILE@' || die "'@OTHER_FILE@' not found, this script must be run from the project base directory"""",
-                Map("OTHER_FILE" -> file.toString))
-        })
+    // Generate shell script that calculates path to project directory from it's own path.
```
Comment (havocp) [inline src/main/scala/com/typesafe/sbt/SbtStartScript.scala]: s/it's/its/ ;-)

**Comment by havocp** [comment]
Looks reasonable to me, if you've tested it running both from the project dir and not from it, I'll merge it in. Let me know - thanks!

**Comment by kriomant** [comment]
It works, here is test session using my project:

```
Projects/buketan on master  !??
$ sbt cmdline/stage
...
[success] Total time: 6 s, completed 03.03.2013 21:32:28
Projects/buketan on master  !??
$ cmdline/target/start
too few arguments
Use '--help' for help.
Projects/buketan on master  !??
$ cd ..
Projects $ buketan/cmdline/target/start
too few arguments
Use '--help' for help.
```

**Comment by havocp** [comment]
thanks! I will try to remember to push a new release soon

**Comment by havocp** [comment]
Would you mind clicking through http://www.typesafe.com/contribute/cla ? it just uses your github account.

**Comment by kriomant** [comment]
Done.

**Comment by kriomant** [comment]
When is next release expected? My project depends on this feature.

**Comment by havocp** [comment]
Just uploaded a 0.7.0, let me know if it is broken.

**Pull request merged**